### ISSUE TITLE
Tab Completion?

### ISSUE THREAD

**Issue by Natfan** [jcplaboratory/rashell#8]
Doesn't seem like tab completion has been introduced, unless I'm missing something.

As there is not yet version numbering, I'm on `commit #56`.

**Comment by Natfan** [comment]
Might wanna give this the `enhancement` tag as it isn't an issue, per se.

**Comment by AnderssonPeter** [comment]
If you do implement this it would be nice to get a preview like in fish on Linux!

**Comment by cedroid09** [comment]
Yes, we will implement it. Thanks for your suggestion. I will definitely look into it. :)
